### DISCUSSION TITLE
Ottawa - Web Application Developer (Student) - OPBO - $16.99 - $ 25.52

### DISCUSSION THREAD

**Discussion by richie119** [richie119/White_Crane_Academy#12] [General]
Web Application Developer (Student) (23-PB-14)
==============================================
[https://www.pbo-dpb.ca/en/careers--carrieres/23-PB-14](url)

The **Office of the Parliamentary Budget Officer (OPBO)** is an independent and non-partisan entity providing stimulating and rewarding work.  

The law mandates us to support Canadian democracy by providing Parliament with analysis that raises “the quality of parliamentary debate and promotes greater budget transparency and accountability.” This includes costing election proposals made by political parties. 

The PBO is supported by a dynamic team of highly specialized analysts, advisors, and professionals. We are currently searching for student candidates to support our work as a Web Application Developer_._ 

The Web Application Developer (Student) designs, writes, and tests code for complex web applications that are built using the Laravel and Vue.js frameworks. 

* * *

**STUDENT RECRUITMENT**  
**23-PB-14** 

**Web Application Developer (Student)** 

**Determinate – Part time**  

**Note:** This selection process may also be used to establish a pool of prequalified candidates that could be used to staff determinate and indeterminate positions at the Office of the Parliamentary Budget Officer that require similar knowledge or competencies. 

**$16.99 - $ 25.52  
****(**[Student rates of pay – Effective May 1, 2022 - Canada.ca](https://www.canada.ca/en/treasury-board-secretariat/services/pay/rates-pay/student-rates-pay.html)) 

**Open to**: Canadian Citizens and permanent residents who are legally able to work in Canada. Consideration will be given to applicants with a valid work permit at time of appointment for determinate employment only. The validity of the work permit must be for a period of no less than one (1) year from the date of appointment. 

**Work Location**: Ottawa, Ontario 

**Closing date****: November 27, 2023** – 23:59 Eastern Standard Time 

* * *

**ESSENTIAL QUALIFICATIONS** 

**Various Language requirements:** 

*   English or French essential 

**Education:** 

The candidate must be: 

*   _Enrolled for the entire duration of employment in a program leading to the completion of a diploma or a degree in computer science, engineering, communications, business, science or in a similar field from a recognized university or college._ 

*   this includes persons with disabilities considered to have full-time status by their academic institution, and participants in adult secondary-level education and retraining programs under the authority of an accredited academic institution 

**Knowledge, Abilities and Competencies:** 

*   Knowledge of software testing principles and market trends of application testing tools and technologies. 
*   Knowledge of modern software development processes, principles, best practices and common tooling (Git, Visual Studio Code, etc.). 
*   Knowledge of cybersecurity development and cloud infrastructure maintenance best practices. 
*   Ability to develop, test and maintain multi-module/complex web applications using PHP 8+, Laravel, PHPUnit and Pest. 
*   Ability to develop, test and maintain multi-module/complex web applications using ECMAScript, Node.js, Vue.js, Vite, Vitest. 
*   Ability to communicate effectively orally and in writing. 
*   Ability to navigate ambiguity and set clear priorities in a rapidly evolving agile environment. 
*   Analytical thinking 
*   Professionalism 
*   Organizational Skills 

**Condition of Employment:** 

*   Successful pre-employment enhanced reliability security screening 

**Additional Information** **–** **_please read carefully_****:** 

*   You **must include a cover letter** with your CV, indicating how you meet the education criteria of the position and how you could help fulfill our mandate. 
*   When submitting your application, please confirm that you are either a Canadian citizen or permanent resident or in possession of a valid work permit.  If you will be eligible to work in Canada but have not yet obtained a work permit, please confirm in cover letter.  Proof of Canadian citizenship, permanent residency or valid work permit will be required prior to appointment. 
*   If you are currently employed in the Public Service of Canada, please identify your employment status (_i.e.,_ indeterminate, determinate, student, casual), substantive group and level, and current second language evaluation results in your application. 
*   Depending on the number of applications, a top-down selection will most likely be used at the screening stage of the process. Selected applicants will be invited to complete a case study during the week of **November 27, 2023**. Interviews will be scheduled **December 5-8, 2023**. Only those selected for further consideration, if any, will be contacted. 
*   Satisfactory references and proof of education and academic enrollment are essential conditions of employment. 
*   We are committed to providing an inclusive and barrier-free work environment, starting with the hiring process. If you need to be accommodated during any phase of the evaluation process, please use the contact information below. All information received in relation to accommodation will be kept confidential.  
*   During this staffing process, masculine generic refers to both binary and non-binary genders and is used for conciseness purposes only. 

* * *

Please quote **Staffing Process 23-PB-14** and send us your C.V. along with your cover letter by email:  

[pbohr@parl.gc.ca](mailto:pbohr@parl.gc.ca) 

**Questions?** Contact Human Resources at 613-992-8026 or [pbohr@parl.gc.ca](mailto:pbohr@parl.gc.ca)   

**We thank all those who apply.**

**Comment by richie119** [comment]
![](/cms/bitmap/Yves_Website-transparentbg.png)

Yves Giroux
===========

Parliamentary Budget Officer

[pbo-dpb@parl.gc.ca](mailto:pbo-dpb@parl.gc.ca)

[613-992-8026](tel:+16139928026)

Yves Giroux was appointed by the Governor in Council to serve as Parliamentary Budget Officer (PBO), effective September 4, 2018.

Mr. Giroux entered the federal public service in 1995. Over the course of his 23-year career, he assumed progressively senior level positions and increased responsibilities, joining the executive ranks in 2003.

Mr. Giroux has in-depth tax, fiscal and financial, and social policy knowledge and expertise, and considerable experience with stakeholder relations, and the management of large teams and multi-million dollar budgets. For more than 20 years, he has been closely involved in the federal Budget process in various capacities and is highly regarded as an expert on federal budget making and the Government’s expenditure system.

Since 2015, Mr. Giroux has been serving as Assistant Commissioner and Chief Data Officer, Strategy and Integration Branch, with the Canada Revenue Agency. Prior to this, he was Director, Operations, Liaison Secretariat for Macroeconomic Policy at the Privy Council Office, for four years. From 2005 to 2011, he served as Director and Senior Chief, Social Policy Division, Finance Canada.

Mr. Giroux is a graduate of the Université de Montréal, where he earned a Master’s and a Bachelor degree, both in Economics. He is married and has two adult children.

**Comment by jovialcore** [comment]
Hi. Is this position still available?
